fs/install

```diff
@@ -152,7 +152,7 @@ if [ "$BASEBOARD" = "trogdor" ]; then
 	chroot $DESTDIR systemctl enable rmtfs
 fi
 
-$BOOTFW_INST_PKGS
+chroot $DESTDIR $FS_INST_PKGS $BOOTFW_PKGS
```

Comment:
it's `FS_INST_PKG`, did you even test this change?

Reply:
I pushed a commit fixing the typo, sorry about that! 

Unfortunately I didn't as I only have 1 laptop for testing and I have a project due tomorrow so I didn't have time to spend the 30-40 minutes reinstalling to test so I'm sorry for the multiple commits fixing typos, I figured it would be a simple fix with the original commit, which I did test and is running on my laptop right now